cluster/examples/cache/rediscluster/aws/resource-class.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     example: "true"
 specTemplate:
   writeConnectionSecretsToNamespace: crossplane-system
-  providerRef:
+  providerConfigRef:
```

Comment:
Did we update classes to use `providerConfigRef`? I thought we only did managed resources

Reply:
Oh, that's right. Will update the PR. Thanks for catching!

Reply:
@hasheddan updated